### DISCUSSION TITLE
Dbus battery White screen

### DISCUSSION THREAD

**Discussion by Fraser86** [Louisvdw/dbus-serialbattery#433] [General]
I am asking for help 
I tried to install Dbus battery @Louisvdw on My PI with a touchscreen
But after reboot, I have a white screen
I have tried to use disable command but the same white screen. 
I am a noob but I have managed ssh and got this far. I'm stuck and do not know how to restore the system

**Comment by Louisvdw** [comment]
Hi @Fraser86 

Your issue would be that you are not on the latest firmware, or perhaps on one of the newer beta firmwares for the GX device.
Log into ssh again and run this command to remove the cell voltage display. 
>cp -f /opt/victronenergy/gui/qml/PageBattery.qml.backup /opt/victronenergy/gui/qml/PageBattery.qml

Reboot and it should be working again as before.

**Reply by Fraser86** [reply]
Thanks Louis  I will try this tonight. Do you mean Venus version? Or the version of dbus? I am on venus 2.8.

**Comment by Fraser86** [comment]
I have tried entering the command above in ssh and rebooted twice but I still only get a white screen
I first see Victron logos and then nothing but a white screen.

 @Louisvdw

**Reply by Cyberb29** [reply]
Update to a firmware between (2.90 - 2.92)

**Comment by Louisvdw** [comment]
You can do command line firmware updates for the GX device in SSH by running this command and following the prompts

>/opt/victronenergy/swupdate-scripts/check-updates.sh

This will also reset the driver, but if you have not run the disable command the driver will reinstall itself again, but this time it should work with the gui changes.

**Reply by Fraser86** [reply]
Hey louis. 

Sorry for delay. 
Before reading your last message, as I was trying to resolve the issue my SD card got extremely hot. I decided to install V3 on a new sd. Got it running and ran the script . However I am collecting Usb data, from my engine room and using Usb to network cable adaptors. These then go into my Pi. While removing my touch screen I accidently damaged this and need to either solder it together again or replace. Then I will be able to see it its worked. 
Q, i should not see anything until the usb is plugged in, unlike other BMS's in gui mods? Is that correct?

**Comment by Fraser86** [comment]
Hey @Louisvdw 

All up and running on version3. Thanks for your help. 
I ran the scanning and it found my BMS. 

Only 
top section voltage 
Bottom section Voltage 
Mid point deviation 

Are not showing at all. 
Is this normal for jkbms

All the best
Fraser
